actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -84,9 +84,9 @@ def self.define_self_closing_element(name, code_generator:, method_name: name.to
             batch.push(<<~RUBY)
               def #{method_name}(content = nil, escape: true, **options, &block)
                 if content || block
-                  tag_string("#{name}", content, escape: escape, **options, &block)
+                  tag_string("#{name.name}", content, escape: escape, **options, &block)
```

Comment:
```suggestion
                  tag_string("#{name.name}", content, options, escape: escape, &block)
```
?

Reply:
Fixed and added a test to cover this case because it was not previously :+1: